back/lib/participation_method/base.rb

```diff
@@ -133,11 +130,11 @@ def supports_permitted_by_everyone?
       false
     end
 
-    def include_data_in_email?
-      true
+    def supports_answer_visible_to?
```

Comment:
What is the difference between this and public visibility?

Reply:
I'm not sure, I also just left this untouched. In the UI schema, this is one of the attributes that is sometimes set, to specify if everyone can see the corresponding field or only admins.